### ISSUE TITLE
Decoder

### ISSUE THREAD

**Issue by MertCokelek** [MertCokelek/VisualSaliencyTransformer#3]
TokenTransformer
- nn.LayerNorm
- nn.Linear
- nn.GeLU
- token_transformerEncoder
  - asd
    - Block
    - nn.LayerNorm
  - saliency - contour_token_inference
    - nn.Linear
    - nn.Dropout
    - nn.Sigmoid
 
Decoder
- decoder_module - token_trans
  - nn.LayerNorm
  - nn.GeLU
  - nn.Linear
  - nn.Fold
